src/class/object_genome_builds.php

```diff
@@ -115,6 +116,33 @@ function prepareData ($zData = '', $sView = 'list')
 
         if ($sView == 'list') {
             $zData['created_date_'] = substr($zData['created_date'], 0, 10);
+
+        } elseif ($sView == 'entry') {
+            global $_DB;
+
+            $sSlash = ($zData['column_suffix']) ? '/' : '';
+
+            $iPercentComplete = $_DB->query(
+                'SELECT (mapped_variants / all_variants) * 100 FROM
+                    (SELECT COUNT(id) AS mapped_variants FROM lovd_variants
+                     WHERE `VariantOnGenome/DNA' . $sSlash . $zData['column_suffix'] . '` IS NOT NULL) mv,
+                    (SELECT COUNT(id) AS all_variants FROM lovd_variants) v')->fetchAllColumn()[0];
```

Comment:
A `fetchAllColumn()` followed by a `[0]` is much less efficient than just a `fetchColumn()`.